Automattic/wp-calypso
# Add a google sheets connection

#### Changes proposed in this Pull Request

* This creates a new connection to Google Sheets from /marketing/connections
* Needs D35254
* This has no useful function yet

#### Testing instructions



* Checkout this branch
* Apply the patch D35254
* Go to http://calypso.localhost:3000/marketing/connections
* Select a site
* Scroll down to Google Sheets and click connect
* You should see a success message and your Google avatar below the connection
* Click disconnect
* You should see another success message

Screenshots:
<img width="1080" alt="Screenshot 2019-10-22 at 13 44 19" src="https://user-images.githubusercontent.com/275961/67286941-1ee2cb80-f4d2-11e9-8675-0d3a08512f71.png">
<img width="1068" alt="Screenshot 2019-10-22 at 13 44 08" src="https://user-images.githubusercontent.com/275961/67286943-1ee2cb80-f4d2-11e9-8065-7ab6c02bf6e4.png">
